From 4ee6b087b4526e652fdf49142e07505b867097e7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 5 May 2024 14:01:26 -0700
Subject: [PATCH] Revert "bindings: set up modern bindings also when binary's
 name starts with "e""

This reverts commit 580eaf29d60734133c11f4ddb4b6aa06cdf33fdc.
---
 src/nano.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/nano.c b/src/nano.c
index 973054f0..dbdbf5db 100644
--- a/src/nano.c
+++ b/src/nano.c
@@ -1846,8 +1846,6 @@ int main(int argc, char **argv)
 	/* If the executable's name starts with 'r', activate restricted mode. */
 	if (*(tail(argv[0])) == 'r')
 		SET(RESTRICTED);
-	else if (*(tail(argv[0])) == 'e')
-		SET(MODERN_BINDINGS);
 
 	while ((optchr = getopt_long(argc, argv, "ABC:DEFGHIJ:KLMNOPQ:RS$T:UVWX:Y:Z"
 				"abcdef:ghijklmno:pqr:s:tuvwxy!%_0/", long_options, NULL)) != -1) {
-- 
2.44.0

